tests/base.py

```diff
@@ -380,10 +382,6 @@ def timedelta_formatted(self, dtime, days=0):
     def is_incremental(self, stream):
         return self.expected_metadata()[stream][self.REPLICATION_METHOD] == self.INCREMENTAL
 
-    def dt_to_ts(self, dtime):
-        for date_format in self.DATETIME_FMT:
-            try:
-                date_stripped = int(time.mktime(dt.strptime(dtime, date_format).timetuple()))
-                return date_stripped
-            except ValueError:
-                continue
+    def dt_to_ts(self, dtime, format):
```

Comment:
Add function comment here.

Reply:
Added comment for the function.

Reply:
Removed comment over the function and added docstring.